doc/src/sphinx/Metrics.rst

```diff
@@ -204,10 +208,10 @@ These stats pertain to the HTTP protocol.
   count as 5XX for this counter.
 
 **time/<statusCode>**
-  Metric on duration per Http status code.
+  A histogram on duration per Http status code.
```

Comment:
sorry, should've noticed previously, please add that these two histograms are in milliseconds.